wanna_go_list.html

```diff
@@ -0,0 +1,441 @@
+                </defs>
+                <g transform="translate(-99 -36)">
+                    <use fill="#0d215f" xlink:href="#a"/>
+                </g>
+            </svg>
+
+        </a>
+
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarTogglerDemo02" aria-controls="navbarTogglerDemo02" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+
+        <div class="collapse navbar-collapse justify-content-end ml-md-4" id="navbarTogglerDemo02">
+
+            <form class="form-inline mt-md-2 mr-xl-5">
+                <input class="form-control mx-lg-3" type="text" placeholder="Search by location or technology...">
+            </form>
+
+            <ul class="nav navbar-nav justify-content-end pr-4">
+                <li class="nav-item">
+                    <a class="nav-link " href="#">Home</a>
+                </li>
+
+                <li class="nav-item">
+                    <a class="nav-link" href="#">Suggest a conference</a>
+                </li>
+
+                <li class="nav-item dropdown">
+                    <a class="nav-link dropdown-toggle active" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                        <img class="mr-1" src="images/superKalo.jpg" width="28" height="28" alt="profile picture">Kaloyan
+                    </a>
+
+                    <div class="dropdown-menu">
+                        <a class="dropdown-item" href="#">Profile Settings</a>
+                        <a class="dropdown-item" href="#">My Subscriptions</a>
+                        <a class="dropdown-item" href="#">Wanna go list</a>
+                        <a class="dropdown-item" href="#">Log out</a>
+                    </div>
+                </li>
+
+            </ul>
+        </div>
+    </nav>
+
+
+    <div class="container mx-auto pt-5 pb-5">
+        <h2 class="text-center mb-5">Wanna Go List</h2>
+
+        <div class="mb-5">
+            <h4 class="mb-3">Last <span class="text-danger">1</span> from all Past conferences</h4>
```

Comment:
Why is this text sooo dangerous?

Reply:
Because text-danger color is changed to the red color we use in the project.